docs/_config.yml

```diff
@@ -0,0 +1,13 @@
+title: Saros
+subtitle: Distributed Party Programming
```

Comment:
`jekyll serve` wouldn't serve the site directly:

```
YOUR SITE COULD NOT BE BUILT:
------------------------------------
No repo name found. Specify using PAGES_REPO_NWO environment variables,
'repository' in your configuration, or set up an 'origin' git remote pointing
to your github.com repository.
```

It worked after adding:

```diff
+ repository: saros-project/saros
```

Reply:
In my test, the command works after installing git. I assumed that git is available. Does this error occur with a git installation?

Reply:
I will add the repository information in order to avoid this error